<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="2" failures="1" errors="0" time="4.502">
  <testsuite name="API Tests" errors="0" failures="1" skipped="0" timestamp="2024-05-30T19:02:58" time="4.314" tests="2">
    <testcase classname="API Tests GET /product - should return all products" name="API Tests GET /product - should return all products" time="0.702">
    </testcase>
    <testcase classname="API Tests POST /products - should create a new customers" name="API Tests POST /products - should create a new customers" time="0.036">
      <failure>Error: expect(jest.fn()).toHaveBeenCalledWith(...expected)

Expected: 201
Received: 500

Number of calls: 1
    at Object.toHaveBeenCalledWith (C:\Users\ambet\OneDrive\Documents\EPSI\Semestre II\MSPR\ptk_products\tests\api.test.js:62:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)</failure>
    </testcase>
  </testsuite>
</testsuites>